doc/rados/troubleshooting/troubleshooting-mon.rst

```diff
@@ -218,43 +222,45 @@ What if the state is ``probing``?
 
 What if state is ``electing``?
 
-  This means the monitor is in the middle of an election. These should be
-  fast to complete, but at times the monitors can get stuck electing. This
-  is usually a sign of a clock skew among the monitor nodes; jump to
-  `Clock Skews`_ for more infos on that. If all your clocks are properly
-  synchronized, it is best if you prepare some logs and reach out to the
-  community. This is not a state that is likely to persist and aside from
+  This means the monitor is in the middle of an election. With recent Ceph
+  releases these typically complete quickly, but at times the monitors can
+  get stuck in what is known as an *election storm*. This can be
+  an artifact clock skew among the monitor nodes; jump to
+  `Clock Skews`_ for more information. If all your clocks are properly
+  synchronized, you should search the mailing lists and tracker.
+  This is not a state that is likely to persist and aside from
   (*really*) old bugs there is not an obvious reason besides clock skews on
-  why this would happen.
+  why this would happen.  Worst case, if there are enough surviving mons,
+  down the problematic one while you investigate.
 
 What if state is ``synchronizing``?
 
-  This means the monitor is synchronizing with the rest of the cluster in
-  order to join the quorum. The synchronization process is as faster as
-  smaller your monitor store is, so if you have a big store it may
-  take a while. Don't worry, it should be finished soon enough.
+  This means the monitor is catching up with the rest of the cluster in
+  order to join the quorum. Time to synchronize is a function of the size
+  of your monitor store and thus of cluster size and state, so if you have a
+  large or degraded cluster this may take a while.
 
-  However, if you notice that the monitor jumps from ``synchronizing`` to
+  If you notice that the monitor jumps from ``synchronizing`` to
   ``electing`` and then back to ``synchronizing``, then you do have a
-  problem: the cluster state is advancing (i.e., generating new maps) way
-  too fast for the synchronization process to keep up. This used to be a
-  thing in early Cuttlefish, but since then the synchronization process was
-  quite refactored and enhanced to avoid just this sort of behavior. If this
-  happens in later versions let us know. And bring some logs
+  problem: the cluster state may be advancing (i.e., generating new maps)
+  too fast for the synchronization process to keep up. This was more common
```

Comment:
s/This was more common/This was a more common/

This phrase modifies the coming noun "thing", so we need the "a" in here.

Reply:
ack